src/ucp/wireup/wireup_cm.c

```diff
@@ -1307,9 +1364,14 @@ ucp_ep_server_init_priv_data(ucp_ep_h ep, const char *dev_name,
     ucp_context_dev_tl_bitmap(worker->context, dev_name, &ctx_tl_bitmap);
     ucp_tl_bitmap_validate(&tl_bitmap, &ctx_tl_bitmap);
 
-    status = ucp_cm_ep_priv_data_pack(ep, &tl_bitmap, UCS_LOG_LEVEL_ERROR,
-                                      sa_data_version, (void**)data_buf_p,
-                                      data_buf_size_p, ep_init_flags);
+    status = ucp_cm_ep_priv_space_is_enough(ep, &tl_bitmap, UCS_LOG_LEVEL_ERROR,
+            ep_init_flags, sa_data_version);
+    if (status != UCS_OK) {
+        goto out;
+    }
```

Comment:
why needed here? ucp_cm_ep_priv_data_pack is already checking the size

Reply:
@yosefe In this PR,  if ```ucp_cm_ep_priv_space_is_enough``` return ```UCS_OK```, ```ucp_cm_ep_priv_data_pack``` should pack the meta data successfully or it'll use [ucs_error](https://github.com/openucx/ucx/pull/8844/commits/9646298c2a72c615804e7d74c0585f032f28a9fd#diff-c0cb7d0175f5028cd00c2505391827822efb64f299314f5958e2bc0f823f3698R362) to report the error directly.